core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt

```diff
@@ -929,22 +933,27 @@ abstract class CoreReaderFragment :
         hideTabSwitcher()
         return FragmentActivityExtensions.Super.ShouldNotCall
       }
+
```

Comment:
Why have you added these spaces that are not relevant to this PR? Please revert these changes.

Reply:
It was showing a lint error to me, should I keep it?

Reply:
Okay, I got it, it is showing a white space error, Keep these changes it would lead to lint failure in future in updated gradle version.